packages/core/src/render3/instructions.ts

```diff
@@ -321,13 +321,15 @@ export function getOrCreateNodeInjector(): LNodeInjector {
  * @param index Index of the element in the data array
  * @param nameOrComponentDef Name of the DOM Node or `ComponentDef`.
  * @param attrs Statically bound set of attributes to be written into the DOM element on creation.
+ * @param localName A name under which a given element is exported.
```

Comment:
why is this a separate parameter? Why don't we just encode it into the attributes like so:
```
['#localName': '#']
```

Or some variation on that? This would also simplify querying?

Reply:
Hmm, but I would either had to assume that it is stored as a first attribute _and_ / _or_ test for name (is it starting with `#`?) OR iterate over the entire attributes array _and_ test for presence of `#`. 

IMO the current proposal is much simpler for covering functionality in the current Angular.